jjb/jobs/qpc-PILOT-nightly-automation.yaml

```diff
@@ -3,18 +3,32 @@
     node: 'f27-os'
     triggers:
         - timed: '@midnight'
+    properties:
+      - build-blocker:
+        blocking-jobs:
+          - 'qpc-.*nightly-automation'
+        queue-scanning: 'ALL'
```

Comment:
Not sure about this yet. The all looks for build in queue which are on all states such as pending (when waiting for a slave to show). My intention to use all here is to give nightly automation more priority and if it is queue this will wait.